### DISCUSSION TITLE
Win版本的PAG导出到100%的时候就会提示失败

### DISCUSSION THREAD

**Discussion by bboyban** [Tencent/libpag#2620] [Q&A] [answered]
每次都是卡在100%，随后就会提示失败。重装了几次PAG都不行
![1](https://github.com/user-attachments/assets/1406652e-aa9e-4747-88f9-3c9a83e9eabf)
![2](https://github.com/user-attachments/assets/b38dcaed-9f87-4299-b8d7-4d834522d9a3)

**Comment by jinwuwu001** [comment]
请问该问题出现的环境是否还在，若还在请帮忙确认如下问题
如下图所示，输入%APPDATA%\H264EncoderTools
<img width="421" alt="1" src="https://github.com/user-attachments/assets/bcc6e090-ea08-4777-8ca0-dae49fa65865" />
然后回车进入对应文件夹，检查下该文件夹下是否存在名为H264EncoderTools.exe的可执行程序
<img width="710" alt="2" src="https://github.com/user-attachments/assets/9efa31fd-fc56-4de1-b325-0561608d2575" />

若不存在且此时AE导出bmp文件失败，请在此评论回复下便于确认问题原因

**Reply by bboyban** [reply]
你好，按照你的方法访问后，不存在H264组件。且导出失败。
![微信截图_20250106171433](https://github.com/user-attachments/assets/3c64b546-7edc-4c32-9e95-217602a61b66)

**Reply by jinwuwu001** [reply, marked as answer]
好的，上述验证说明PAGViewer在进行ae插件安装时没有正确安装264编码工具，该问题是偶现的，目前正在处理中，后续版本会修复，可通过如下两种方式暂时规避该问题
1、可以在PAGViewer中多次安装ae插件，确保%APPDATA%\H264EncoderTools下存在编码工具后再进行pag导出
2、在PAGViewer安装目录下找到可执行程序PAGViewer/H264EncoderTools.exe，将其复制至%APPDATA%\H264EncoderTools，然后再进行pag文件导出

**Reply by bboyban** [reply]
已解决，感谢你的专业解答。
